scripts/blogView.js

```diff
@@ -0,0 +1,17 @@
+var blogView = {};
+
+blogView.setTeasers = function() {
+  $('.blog-body *:nth-of-type(n+2)').hide();
+
+
+  $('blogs').on('click', 'a.read-on', function(e) {
```

Comment:
You don't have any element called 'blogs'. You probably want to attach this to .read-on instead.